karma.conf.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./config/karma.config.js');
```

Comment:
You guys are missing the '.' in front of config. We have a config directory not a .config directory.